anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py

```diff
@@ -46,8 +53,8 @@ def motorcycle_accidents_vs_all_accidents(
         ).label(location_label)
 
         vehicle_label = "vehicle"
-        vehicle_other = "אחר"
-        vehicle_motorcycle = "אופנוע"
+        vehicle_other = VehicleCategory.OTHER.get_english_display_name()

```

Comment:
This is a bit confusion but VehicleCategory.OTHER is referred as "CBS Other category", but here it's referred as all other vehicles. @ziv17 do you think we should add a different constant here?

Reply:
@atalyaalon , you are right. The label here should be `all other vehicles`, or כל האחרים
right?